libs/backend/src/initialize_system_audio.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  BooleanEnvironmentVariableName,
+  isFeatureFlagEnabled,
+} from '@votingworks/utils';
+import { execFile } from './exec';
+
+/**
+ * Configures system audio settings for audio-enabled precinct machines.
+ *
+ * VVSG 2.0 requires setting audio volume within an explicit loudness
+ * range and in gain increments of <= 10 dB. This is easier to do if we set the
+ * system volume to a fixed level and adjust the amplification for audio clips
+ * during playback by explicit decibel amounts, instead of adjusting the system
+ * volume by percentage amounts.
+ *
+ * Setting the system volume to 100% allows us to limit how much positive gain
+ * we apply to audio and avoid signal distortion at higher volumes.
+ */
+export async function initializeSystemAudio(): Promise<void> {
+  // Allow skipping when there's no sound card available (e.g. for integration
+  // tests in CI environments).
```

Comment:
If the only context that we can think of where we need to skip system audio setup is integration tests, we could use `isIntegrationTest` instead of adding a feature flag:

https://github.com/votingworks/vxsuite/blob/7d4143febe0f345c44ad2a97fbeb1f44f0a681bf/libs/utils/src/environment_variable.ts#L258-L265

I'm okay with the explicitness of a feature flag too, though

Reply:
Oh, perfect! Exactly what I was looking for, but not in the right places, apparently 😆  - thanks